spec/models/active_record/localizer_test_model.rb

```diff
@@ -11,13 +11,13 @@ class LocalizerTestModel < ActiveRecord::Base
 
 describe 'localized state names' do
   before(:all) do
-    I18n.load_path << 'spec/en.yml'
-    I18n.default_locale = :en
+    I18n.load_path << 'spec/localizer_test_model_new_style.yml'
     I18n.reload!
   end
 
   after(:all) do
-    I18n.load_path.clear
```

Comment:
Calling `load_path.clear` leads to errors in all specs which will be run after that (because all default translations that are provided by `activerecord`, `mongoid`, etc. are removed).

So I moved all translations that necessary just for this spec-file ([these lines](https://github.com/aasm/aasm/pull/689/files#diff-91a325edd28df1a02e0ffe9f85ac31cfL7-L9)) into separate yml-file, so I can safely add/remove it to/from `load_path` without removing default translations for `activerecord` and `mongoid`.